aea/test_tools/docker_image.py

```diff
@@ -114,7 +116,6 @@ def wait(self, max_attempts: int = 15, sleep_rate: float = 1.0) -> bool:
         :param sleep_rate: the amount of time to sleep between different requests.
         :return: True if the wait was successful, False otherwise.
         """
-        return True  # pragma: no cover
```

Comment:
@angrybayblade somehow removed in this PR, I had put it back in the previous before merging that into here again